docs/api-guide.md

```diff
@@ -250,6 +250,18 @@ You can get table metadata as follows:
 TableMetadata tableMetadata = admin.getTableMetadata("ns", "tbl");
 ```
 
+### Upgrade the environment to support the latest ScalarDB API
+
+You can upgrade the ScalarDB environment to support the latest version of the ScalarDB API. Typically,
```

Comment:
[trivial] Maybe this is a matter of taste, but

> you will be requested

"you will need" is a bit clearer?

Reply:
I agree; it sounds better. Thanks.